### PR TITLE
Arabic Translation  added

### PR DESCRIPTION
**Pull request by AhmadAymanA99**

### CODE REVIEW

**Comment by rugk** [comment]
Hi @AhmadAymanA99,
thanks for your first contribution to this project! :tada: :+1: 🏅
I hope you'll like this project and enjoy hacking on it… :smiley:

Thanks for your translation.
Please adjust the indentation to two spaces. This is [a known issue](https://github.com/Lusito/web-ext-translator/issues/30).) Otherwise I'll soon do the PR by myself to get it merged.

Also notice there are more things to translate like the add-on listings on AMO (addons.mozilla.org) etc. [See the contributing guide for more information.](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translation-of-add-on-listings)

Finally, please subscribe to #138 to get notified for new updates on translations etc. :smiley:

File: src/_locales/ar/messages.json [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+      "description": "The credit text for the translator. See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on for how to translate this.",
+      "placeholders": {
+        "translators": {
+          "content": "$1",
+          "example": "<a href=\"https://github.com/AhmadAymanA99\">@AhmadAymanA99</a>"
+        }
+      }
+    },
+    "translatorLink": {
+      "message": "https://github.com/rugk",
```
Comment (rugk) [inline src/_locales/ar/messages.json]:
```suggestion
      "message": "https://github.com/AhmadAymanA99",
```

You likely want to link to yourself.

File: src/_locales/ar/messages.json [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+      "message": "تمكين وضع التفحيص",
+      "description": "This is an option shown in the add-on settings."
+    },
+    "optionDebugModeDescr": {
+      "message": "هذا مفيد فقط للحصول على ناتج تسجيل أكثر تفصيلاً في وحدة التحكم للإبلاغ عن الأخطاء ، إلخ",
+      "description": "The description shown for the debug mode (optionDebugMode)."
+    },
+  
+    "translatorCredit": {
+      "message": "This add-on has been translated into Arabic by Ahmad Ayman $LINK$ !",
```
Comment (rugk) [inline src/_locales/ar/messages.json]:
```suggestion
      "message": "This add-on has been translated into Arabic by Ahmad Ayman ($LINK$)!",
```

* remove space (it is not needed is it?)
* only link part inside of brackets

(see [doc](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on) for reference)

File: src/_locales/ar/messages.json [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+          "example": "<a href=\"https://github.com/AhmadAymanA99\">@AhmadAymanA99</a>"
+        }
+      }
+    },
+    "translatorLink": {
+      "message": "https://github.com/rugk",
+      "description": "The link to the translator's GitHub profile."
+    },
+    "translatorUsername": {
+      "message": "(@AhmadAymanA99)",
```
Comment (rugk) [inline src/_locales/ar/messages.json]:
```suggestion
      "message": "@AhmadAymanA99",
```

(see above) brackets are now not linked inside the message

File: src/_locales/ar/messages.json [hunk 4]
```diff
@@ -0,0 +1,429 @@
...
+      "description": "The username that the translator wants to be referred to."
+    },
+  
+    "contributorsThanks": {
+      "message": "Also thanks to $CONTRIBUTORS$.",
```
Comment (rugk) [inline src/_locales/ar/messages.json]: This is not translated… See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translations for how these placeholders are translated, especially [the official doc](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales). Same with things like `contributorsThanksLinkText` below.

**Comment by rugk** [comment]
@AhmadAymanA99 Are you still interested in completing this? If not, that's no problem, just add a notice, so I know what the current status of this PR is.

**Comment by rugk** [comment]
BTW you can click "Resolve conversation" if you did solve what I've requested there.

And if you want to test the changes:
To test out the code, you need to clone the repo locally (with git submodules) and load it into Firefox. Fortunately, that is quite easy without having to install additional tools. [In the contributing guide](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#coding), you can see how to do that.

Let me know if you are content with the result.

File: src/_locales/ar/messages.json [hunk 5]
```diff
@@ -0,0 +1,429 @@
...
+    "ariaMessageError": {
+      "message": "خطأ",
+      "description": "the aria label to label the message box as an error message box"
+    },
+    "ariaMessageWarning": {
+      "message": "تحذير",
+      "description": "the aria label to label the message box as an warning message box"
+    }
+  }
+
```
Comment (rugk) [inline src/_locales/ar/messages.json:429]: AFAIK there is a bracket missing:
```suggestion
}
```

File: CONTRIBUTORS.md [hunk 6]
```diff
@@ -51,3 +51,7 @@
 
 - Ömür Turan ([@omurturan](https://github.com/omurturan))
 - Serkan Algur ([@serkanalgur](https://github.com/serkanalgur))
+
+### Arabic
+
+- Ahmad Ayman([@AhmadAymanA99](https://github.com/AhmadAymanA99))
```
Comment (rugk) [inline CONTRIBUTORS.md:57]: you do miss one space here:
```suggestion
- Ahmad Ayman ([@AhmadAymanA99](https://github.com/AhmadAymanA99))
```

**Comment by rugk** [comment]
@AhmadAymanA99 If you need any pointers/help or have questions, feel free to ask.

**Comment by rugk** [comment]
@AhmadAymanA99 Are you still interested in completing this? If not, that's no problem, just add a notice, so I know what the current status of this PR is.

**Comment by AhmadAymanA99** [comment]
I am interested. Please clarify what you are waiting for.

**Comment by AhmadAymanA99** [comment]
I think I just handled what you asked for.

**Comment by rugk** [comment]
So finally thanks a lot and sorry for the delay when handling this.

Please subscribe to #138 to get notified for new updates on translations etc. :smiley:

**Comment by AhmadAymanA99** [comment]
I want to thank you too. You were very helpful and patient. I learned a lot
from this little experience.
I also wanted to test the extension but couldn't add it locally to firefox.

**Comment by rugk** [comment]
If you still want to test it, feel free to ask any questions.

**Pull request merged**